contrib/build-push-ceph-container-imgs.sh

```diff
@@ -34,11 +34,18 @@ fi
 CONTAINER_REPO_USERNAME=${CONTAINER_REPO_USERNAME:-$DOCKER_HUB_USERNAME}
 CONTAINER_REPO_PASSWORD=${CONTAINER_REPO_PASSWORD:-$DOCKER_HUB_PASSWORD}
 
+OUTFILE=../src/daemon-base/__CRIMSON_PACKAGES__
+if [[ "$FLAVOR" == "crimson" ]] && ! grep -q "crimson" $OUTFILE ; then
+  echo "        ceph-crimson-osd__ENV_[CEPH_POINT_RELEASE]__" >> $OUTFILE
+fi
+
```

Comment:
We shouldn't do this kind of hack.

This means, this won't work outside of this script whereas one can build the container manually without this script.

Reply:
sure! sounds good!